index.html

```diff
@@ -38,6 +38,138 @@ <h1>Hey there. I’m Promise <span class="title">I’m a software developer</spa
           </div>
         </div>
       </section>
+
+      <div class="go-to-works">
+        <a href="#works"><img src="./img/arrow.png" alt="arrow"></a>
+      </div>
+
+      <section id="works" class="grid">
+        <h2>My Recent Works</h2>
+        <hr>
+        <div class="card border">
+          <img src="./img/smartfacedetector.png" alt="smartfacedetector">
+          <div class="project-name">
+            <h4>My Smart Face Detector App</h4>
+          </div>
+          <div class="tech-group">
+            <span class="tech">Nodejs</span>
+            <span class="tech">css</span>
+            <span class="tech">Javascript</span>
+            <span class="tech">html</span>
+          </div>
+          <button class="btn" type="button">See Project</button>
```

Comment:
- [ ] Please apply the hover property for the buttons. In the Figma design, you can find this in the #Buttons section. 👁️ 👍 

![Capture](https://user-images.githubusercontent.com/64173189/133855121-48a9c1d1-2748-461f-a24f-0f3e7606730e.JPG)

Reply:
Fixed.